pkg/roachprod/cloud/gc_aws.go

```diff
@@ -316,3 +325,108 @@ func GCAWSKeyPairs(l *logger.Logger, dryrun bool) error {
 	}
 	return nil
 }
+
+// gc garbage-collects expired clusters, unused SSH key pairs in a
+// single AWS account
+func gc(l *logger.Logger, dryrun bool) error {
+	var (
+		wg            sync.WaitGroup
+		keyPairErr    error
+		combinedError error
+	)
+
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		keyPairErr = gcAWSKeyPairs(l, dryrun)
+	}()
+
+	cld, _ := ListCloud(l, vm.ListOptions{IncludeEmptyClusters: true, IncludeProviders: []string{aws2.ProviderName}})
```

Comment:
In case ListCloud fails, `cld` will be nil and GCClusters will not do anything as there is no vm to garbage collect. Do not want the cronjob to fail because of some intermittent issue for one of the account out of multiple IAM account.